mobile/lib/features/wallet/send/send_screen.dart

```diff
@@ -176,13 +176,14 @@ class _SendScreenState extends State<SendScreen> {
                     ),
                   )),
               const SizedBox(height: 20),
-              const Text("Amount in sats", style: TextStyle(fontWeight: FontWeight.bold)),
+              const Text("Amount in sats (0 to drain the wallet)",
+                  style: TextStyle(fontWeight: FontWeight.bold)),
               const SizedBox(height: 2),
               AmountInputField(
                 controller: _controller,
                 label: "",
-                value: _amount ?? Amount.zero(),
-                enabled: _destination != null && _destination!.amount.sats == 0,
+                value: _amount,
+                enabled: _destination != null,
```

Comment:
🔧 this shouldn't be editable if the amount is set on the invoice or bip21 uri.

Reply:
fixed

Reply:
This is still editable if the amount is set. Isn't it?

Reply:
no, it's not editable, the logic is unchanged to before.